docs/census.rst

```diff
@@ -0,0 +1,78 @@
+Census
+=============
+
+********
+Overview
+********
+
+Connects to the Census API--it has been tested with the ACS and Economic Survey endpoints.
+
+.. note::
+  Authentication
+    The API requires a key that you can get `here <https://www.census.gov/data/developers.html>`_ (click on the box that says "Request a KEY")
+
+
+
+==========
+Quickstart
+==========
+
+To instantiate the Census class, either store your ``CENSUS_API_KEY`` as an environment
+variable or pass it as an argument:
+
+.. code-block:: python
+
+   from parsons import Census
+
+   # First approach: Use API key environment variables
+
+   # In Mac OS, for example, set your environment variables like so:
+   # export CENSUS_API_KEY='MY_CENSUS_KEY'
+
+   census = ActBlue()
+
+   # Second approach: Pass API key as arguments
+   actblue = ActBlue(api_key = 'MY_CENSUS_KEY')
+
+
+
+**Example 1**
+
+.. code-block:: python
+
+  from parsons import Census
+  # these variables tell the API what data we want
```

Comment:
Slightly confusing, by "these variables" do you mean the four variables listed below, or the variable named "variables"? If the latter, maybe put that as a comment on line 47? And maybe have one as an example, like "for example 'B01001_001E` is 'whatever that is'"